pyuvdata/parameter.py

```diff
@@ -326,14 +326,14 @@ def expected_shape(self, uvbase):
         """
         if self.form == "str":
             return self.form
-        elif isinstance(self.form, (int, np.integer)):
+        elif isinstance(self.form, (int, np.int_)):
             # Fixed shape, just return the form
             return (self.form,)
         else:
             # Given by other attributes, look up values
             eshape = ()
             for p in self.form:
-                if isinstance(p, (int, np.integer)):
+                if isinstance(p, (int, np.int_)):
```

Comment:
These lines caught my eye. Any particular reason you removed the more general `np.integer` here? I think the isinstance is fine using the generic base class. Basically it let's users use not 64bit integers as the elements to "form".

Reply:
Just saw this too. I don't think I should have made this change, it should go back to being `np.integer`.